src/transformers/feature_extraction_common_utils.py

```diff
@@ -192,42 +190,19 @@ def to(self, device: Union[str, "torch.device"]) -> "BatchFeature":
         return self
 
 
-class PreTrainedFeatureExtractor:
+class FeatureExtractionSavingUtilsMixin:
     """
-    This is a general feature extraction class for speech recognition.
-
-    Args:
-        feature_size (:obj:`int`):
-            The feature dimension of the extracted features.
-        sampling_rate (:obj:`int`):
-            The sampling rate at which the audio files should be digitalized expressed in Hertz per second (Hz).
-        padding_value (:obj:`float`):
-            The value that is used to fill the padding values / vectors.
+    This is a feature extraction mixin used to provide saving/loading functionality for sequential and image feature
+    extractors.
     """
 
-    def __init__(self, feature_size: int, sampling_rate: int, padding_value: float, **kwargs):
-        self.feature_size = feature_size
-        self.sampling_rate = sampling_rate
-        self.padding_value = padding_value
-
-        self.padding_side = kwargs.pop("padding_side", "right")
-        self.return_attention_mask = kwargs.pop("return_attention_mask", True)
-
-        # Additional attributes without default values
-        for key, value in kwargs.items():
-            try:
-                setattr(self, key, value)
-            except AttributeError as err:
-                logger.error(f"Can't set {key} with value {value} for {self}")
-                raise err
-
     @classmethod
     def from_pretrained(
         cls, pretrained_model_name_or_path: Union[str, os.PathLike], **kwargs
     ) -> "PreTrainedFeatureExtractor":
```

Comment:
If you're removing quotes, might as well do it everywhere :)

```suggestion
    ) -> PreTrainedFeatureExtractor:
```